client/jetpack-connect/authorize.js

```diff
@@ -531,7 +529,10 @@ export class JetpackAuthorize extends Component {
 			return `/start/pressable-nux?blogid=${ clientId }`;
 		}
 
-		return addQueryArgs( { redirect: redirectAfterAuth }, `${ JPC_PATH_PLANS }/${ siteSlug }` );
+		return addQueryArgs(
+			{ redirect: redirectAfterAuth },
+			`${ JPC_PATH_PLANS }/${ urlToSlug( homeUrl ) }`
```

Comment:
If we believe `homeUrl` is reliable as part of the `authQuery`, we should adjust the schema appropriately.

We can either add it to `required` (fail early if the request doesn't include it) or we could make it fall back to `site`, which would be the previous behavior.

Thoughts? @jeherve @georgestephanis

Reply:
Honestly unsure -- I'd be happy to give you a site that demonstrates the issue for you to test against if it helps.

Reply:
Whoops, right in front of our eyes, `home_url` is required by the schema and just needs to be  cleaned up in the transform.

For reference, it's at least as reliable as `site_url`:

https://github.com/Automattic/jetpack/blame/02dc0701044e1238f2605a34d3bec26a1ef3739d/class.jetpack.php#L4508